doc/source/cluster/metrics.md

```diff
@@ -0,0 +1,174 @@
+(collect-metrics)=
+# Collecting metrics
```

Comment:
do we have a link to this page from the observabiltiy tab?

Reply:
<img width="818" alt="Screenshot 2023-06-01 at 9 44 23 PM" src="https://github.com/ray-project/ray/assets/9677264/eab51a3a-3c09-416f-a571-00eeaf6ce265">
There are several cross-links, like this one ^